pkgs/development/libraries/openfst/default.nix

```
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, python3, ncurses, lib, autoreconfHook }:
```

Is `lib` really needed?
```suggestion
{ stdenv, fetchurl, python3, ncurses, autoreconfHook }:
```

---

maybe it was before.

---

It was you who added `lib`, I probably confused you with the suggestion.

---

I meant, I maybe used it before in the expression and removed it later during refactoring. Drop it now.